vue-components/package.json

```diff
@@ -77,26 +76,27 @@
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-promise": "^4.2.1",
     "eslint-plugin-standard": "^4.1.0",
-    "eslint-plugin-vue": "^6.2.2",
+    "eslint-plugin-vue": "^9.17.0",
     "eslint-plugin-vue-root-class": "^0.1.0",
     "geckodriver": "^1.19.1",
     "jest-axe": "^4.1.0",
     "jest-serializer-vue": "^2.0.2",
     "npm-run-all": "^4.1.5",
-    "postcss": "^7.0.35",
-    "postcss-dir-pseudo-class": "^5.0.0",
-    "postcss-inline-svg": "^4.1.0",
-    "postcss-logical": "^4.0.2",
-    "postcss-selector-parser": "^6.0.4",
+    "postcss": "^8.4.27",
+    "postcss-dir-pseudo-class": "^8.0.0",
+    "postcss-inline-svg": "^6.0.0",
+    "postcss-logical": "^7.0.0",
+    "postcss-selector-parser": "^6.0.13",
     "sass": "^1.29.0",
     "sass-loader": "^9.0.3",
     "saucelabs": "^4.5.2",
     "stylelint": "^13.13.1",
     "stylelint-config-sass-guidelines": "^7.1.0",
     "stylelint-config-standard": "^20.0.0",
     "stylelint-use-logical": "^1.1.0",
-    "typescript": "^3.9.7",
-    "vue-template-compiler": "^2.6.12"
+    "typescript": "^4.4.4",
+    "@vue/compiler-sfc": "^2.7.14",
+    "vue-template-compiler": "^2.7.14"
```

Comment:
in principle `"@vue/compiler-sfc` replaces `vue-template-compiler` but it is recommended to leave it if the project uses vue/test-utils. More info. [2nd point of Migration guide](https://v2.vuejs.org/v2/guide/migration-vue-2-7.html#Vue-CLI-webpack).